functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/JobsSpec.groovy

```diff
@@ -396,4 +401,216 @@ class JobsSpec extends SeleniumBase {
             jobCreatePage.addSimpleCommandStep 'echo selenium test', 0
             jobCreatePage.createJobButton.click()
     }
+
+    def "Run job later"(){
+        given:
+        def projectName = "run-job-later-test"
+        JobCreatePage jobCreatePage = page JobCreatePage
+        JobShowPage jobShowPage = page JobShowPage
+        ActivityPage activityPage = page ActivityPage
+        ExecutionShowPage executionShowPage = page ExecutionShowPage
+
+        when:
+        setupProject(projectName)
+        go JobCreatePage, projectName
```

Comment:
same as "ExpandedJobGroupsSpec.groovy" observation, you can upload the job using the API